### ISSUE TITLE
unformatted workspace

### ISSUE THREAD

**Issue by reginasoares** [jdemetra/jdemetra-app#352]
![capture 31-05-2017](https://cloud.githubusercontent.com/assets/11855872/26634957/20de959a-4610-11e7-9402-d654ff9f940b.PNG)
![capture 31-05-2017](https://cloud.githubusercontent.com/assets/11855872/26635006/428cf2a4-4610-11e7-92b7-c58079a2bab7.PNG)

I don't know why this situation happens, but sometimes when I change the specification of one of series the workspace becomes like you can see in the attached image.

Regards
Regina

**Comment by charphi** [comment]
I can't reproduce your bug.
Could you describe the exact steps that leads to the problem?

Could you also give me some information about your system environment like Windows/Java version ?
You can copy/paste this info from Menu>Help>About (do not paste here your user and cache directories).

**Comment by Immurb** [comment]
To reproduce the bug you have to do the following steps:
1. Change anything in the specification of a SaItem in the multidocument
2. Apply the change but don't save it
3. Select a new Specification for the SaItem with Rightclick->Specification-> Select...
4. Answer the question "Do you want to save the unsaved change?" with "Yes"

Then you will have this output.

**Comment by charphi** [comment]
Thanks for the info.

**Comment by reginasoares** [comment]
Dear Phillippe,

Sorry for the late answer, but I have some days out and only now I saw your 
question.



I couldn’t reproduce the situation because I didn’t understand why it 
happened (it dosn’t give any error message), but I saw the answer of Thomas.



Anyway I send you information about your system environment like 
Windows/Java version, but this situation happened before, with other Java 
version and even with other machine.



Thank you for your attention.

Best regards

Regina
